packages/controller/src/dbSDK.ts

```diff
@@ -28,9 +29,9 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
-  timestamp: any;
-  timestamptz: any;
-  uuid: any;
+  timestamp: unknown;
```

Comment:
Shouldn't it actually be number? Will it cause issues somewhere if we change it to number?